chainerio/containers/zip.py

```diff
@@ -29,9 +32,15 @@ def _check_zip_file_name(self, base):
         filename, file_extension = os.path.splitext(self.base)
 
     def _open_zip_file(self, mode='r'):
-        mode = mode.replace("b", "")
+        if self.zip_file_obj is not None:
+            if self.zip_file_obj_pid != os.getpid():
+                self.zip_file_obj = None
+                self.zip_file_obj_pid = None
+                self.zip_file_obj_mode = None
```

Comment:
I do not think you need to save the "mode"

Reply:
`mode` requires to check the mode of previous zip.